### ISSUE TITLE
Invalid Layers

### ISSUE THREAD

**Issue by Cotnap** [MerginMaps/qgis-mergin-plugin#408]
I have used Mergin/Input on a casual basis and it has worked okay but now none of my projects will load the layers. The backdrops appear but no data. I have logged in and out, deleted some projects, got storage down to 30mb, changed data sources but without luck. It used to work so why not now? I get a message saying "There were issues loading the project" and an option for Details. This just tells me the layers are unavailable and invalid, see pic. I have loaded Input into both IoS and Android. I have created a new ID in Mergin and also created a project on the phone which work in QGIS okay but fail to synchronise back to the phone. 

Initial data constructed in QGIS 3.20.0 Odense on a Mac running Monterey. Mergin plugin recently refreshed.

Help!

![52473B3D-DBE8-4192-A5F3-F97D5216E17D](https://user-images.githubusercontent.com/62502979/169536576-1b8f0432-4c3a-446b-bedb-1409d58287f2.png)

**Comment by Cotnap** [comment]
Hi @PeterPetrik - is this an actual bug? If so presumably someone will have a look at it and I can stop banging my head on the wall! Please let me know if i can help in any way.

**Comment by PeterPetrik** [comment]
Can you please upgrade to latest Mergin Maps plugin and see if you have any errors/warnings listed? Looks like your Layer is not in the same folder as your QGIS project.

**Comment by Cotnap** [comment]
Hi Peter
Reinstalled plugin to;

Available version (stable) | 2022.3.2 updated at Wed May 11 09:55:43 2022
No obvious errors.

![Mergin test folder](https://user-images.githubusercontent.com/62502979/172696082-072eddbb-9d02-42c6-b7a9-d58bac3633a0.jpg)

All data/ project files appear to be in same folder as do all other projects apart from the backdrop mapping which is from open source but does appear on the phone!
Also, I created a project on the phone and synced it back to QGIS. I changed the symbology of the survey layer which was enough to create the problem when syncing back to the phone.

**Comment by Cotnap** [comment]
I appear to have found a solution! For anyone who is interested my initial problem seemed to stem from an over zealous bout of file & folder housework. This was compounded trying to reuse some layer symbology. This resulted in a bit of a mess with sub folders within folders as I got increasingly paranoid about losing data. The project data appeared to have the correct sources but there must have been something with the wrong address lurking in the background.
In the end I created an entirely new folder for Mergin files. I opened a existing project and by creating a new project based on the original but in new folder with a different project name they now appear to work okay. It will seem obvious now but I was concerned about losing a lot of data and symbology.
